test/test.js

```diff
@@ -11,12 +11,14 @@ var getSetTimeoutPromise = require('./lib/set-timeout-promise-helper');
 var usbDetect = require('../');
 
 const MANUAL_INTERACTION_TIMEOUT = 10000;
+const USB_TEST_VID = process.env.TEST_VID || 5824; // VID of the tevice used to test
+const USB_TEST_PID = process.env.TEST_PID || 1155;
```

Comment:
We no longer need this `USB_TEST_VID`/`USB_TEST_PID` change

Reply:
it's gone now